tests/integration/test_bayesian_optimization.py

```diff
@@ -31,14 +31,17 @@
 from trieste.bayesian_optimizer import BayesianOptimizer
 from trieste.data import Dataset
 from trieste.models import GaussianProcessRegression
-from trieste.models.optimizer import TFKerasOptimizer
+from trieste.models.keras.data import EnsembleDataTransformer
+from trieste.models.optimizer import TFKerasOptimizer, TrainingData
 from trieste.models.keras.models import NeuralNetworkEnsemble
 from trieste.models.keras.networks import MultilayerFcNetwork
 from trieste.models.keras.utils import get_tensor_spec_from_data
 from trieste.space import Box
 from trieste.utils.objectives import BRANIN_MINIMIZERS, BRANIN_MINIMUM, branin, mk_observer
 
 
+tf.keras.backend.set_floatx('float64')
```

Comment:
This might not be necessary.

Reply:
right, I was wondering about that one as well, was on my to do list to check, lets set it up that way for now as trieste seems to use 64 evereywhere